analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -1383,4 +1384,93 @@ public void Invocation_NullableGetHasValue_UntrackedSymbol()
             x => x.Should().HaveOnlyConstraints(ObjectConstraint.NotNull),  // Because it's int
             x => x.Should().HaveOnlyConstraints(ObjectConstraint.NotNull, NumberConstraint.From(0)));
     }
+
+    [TestMethod]
+    public void Invocation_EnumerableCount_SetsNumberConstraint()
```

Comment:
Checks the first part of the `LearnFrom` for invocation, that learns NumberConstraint for `Enumerable.Count()` based on CollectionConstraint.